flang/lib/Optimizer/Transforms/ArrayValueCopy.cpp

```diff
@@ -353,24 +339,30 @@ void ArrayCopyAnalysis::arrayMentions(
 
   // Process the worklist until done.
   while (!queue.empty()) {
-    auto *operand = queue.pop_back_val();
-    auto *owner = operand->getOwner();
-    if (!owner)
-      continue;
+    mlir::OpOperand *operand = queue.pop_back_val();
+    mlir::Operation *owner = operand->getOwner();
```

Comment:
Is `owner` never null? (It must have happened at one point, hence the test.)

Reply:
According to Mehdi it can't be null

Reply:
OK. Hopefully there is no SIGSEGV that crops up.